bedrock/firefox/templates/firefox/os/mwc-2015-preview.html

```diff
@@ -0,0 +1,180 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Blaze Your Own Path - Mozilla at MWC 2015')}}{% endblock %}
+
+{% block body_id %}firefox-os{% endblock %}
+{% block body_class %}firefox-os{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_os_mwc_2015_preview' %}
+  <!--[if lt IE 8]>
+  {% stylesheet 'firefox_os_mwc_2015_preview_ie7' %}
```

Comment:
No need for this conditional comment since the IE7 hacks were removed (this doesn't seem to output anything now, just an empty CC, so it's not actually breaking anything either).